### PR TITLE
bumped version for 0.8.1 patch release

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

**Comment by hosiet** [comment]
There's also data/debian/changelog. We can simply replace 0.8.0 with 0.8.1 or I can provide a patch with new entry.

**Comment by borgmanJeremy** [comment]
okay I bumped it in the changelog

**Comment by hosiet** [comment]
Please consider applying the following patch:

```patch
diff --git a/data/debian/changelog b/data/debian/changelog
index f62c35c..e06b8f0 100644
--- a/data/debian/changelog
+++ b/data/debian/changelog
@@ -1,17 +1,5 @@
 flameshot (0.8.1-1) unstable; urgency=medium
 
-  * add: added translation:Japanese, Brazilian Portuguese, Serbian, Dutch, Ukrainian, German, Slovak, Basque, Czech, Swedish, Italian, Korean, Dutch
-  * add: allow enter key to copy image to clipboard
-  * add: side panel add thickness slider
-  * add: support for saving as JPG and BMP files
-  * add: allow 45 degree adjustment for some tools
-  * add: add option to close after capture (WARNING: this function is buggy! It may not work well if you are going to copy the image to clipboard! If you encounter problems related to clipboard, please disable this feature in the configuration and try again.)
-  * add: a basic launcher panel.
-  * add: add option to auto copy URL after upload
-  * add: a circle counter tool.
-  * add: replace the blur tool with pixelate tool
-  * add: convert buildsystem from QMake to CMake
-  * add: add launcher action into .desktop file
-  * fix: fix capture after pressing ctrl + S during textarea input
+  * New stable release.
 
- -- Boyuan Yang <byang@debian.org>  Sun, 13 Sep 2020 19:33:20 -0400
+ -- Boyuan Yang <byang@debian.org>  Wed, 23 Sep 2020 20:39:29 -0400
diff --git a/docs/appdata/flameshot.metainfo.xml b/docs/appdata/flameshot.metainfo.xml
index e9bc4f3..56d9ef2 100644
--- a/docs/appdata/flameshot.metainfo.xml
+++ b/docs/appdata/flameshot.metainfo.xml
@@ -5,7 +5,8 @@
   <project_license>GPL-3.0-or-later</project_license>
   <name>Flameshot</name>
   <releases> 
-	  <release version="0.8.1" date="2020-09-23"/>
+    <release version="0.8.1" date="2020-09-23"/>
+    <release version="0.8.0" date="2020-09-19"/>
   </releases>
   <summary>Powerful and simple to use screenshot software</summary>
   <description>
```

**Comment by borgmanJeremy** [comment]
Ok thanks, I applied that patch.

File: data/rpm/flameshot.spec [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Name: flameshot
-Version: 0.8.0
+Version: 0.8.1
 Release: 1%{?dist}
 Summary: Powerful yet simple to use screenshot software
 Summary(eu-ES): Potente pero simple de usar software de capturas
```
Comment (ZetaoYang) [inline data/rpm/flameshot.spec:5]: The changelog in the spec file was not updated.

Comment (borgmanJeremy) [reply data/rpm/flameshot.spec:5]: Do the rpm files need to be re uploaded or is it good enough to just fix on the main branch?

Comment (ZetaoYang) [reply data/rpm/flameshot.spec:5]: No need to re-upload, it's just that the changlog needs to be updated.

**Pull request merged**